aws/resources.go

```diff
@@ -1083,6 +1110,10 @@ func iamUsers(ctx context.Context, a *aws, resourceType string, filters *filter.
 }
 
 func iamUserGroupMemberships(ctx context.Context, a *aws, resourceType string, filters *filter.Filter) ([]provider.Resource, error) {
+	// if both aws_iam_group_membership and aws_iam_user_group_membership defined, keep only aws_iam_group_membership
+	if filters.IsIncluded("aws_iam_group_membership") && (!filters.IsExcluded("aws_iam_group_membership")) {
```

Comment:
This condition is weird ? 
We check if the `aws_iam_group_membership` is included and if it's not excluded (so included) ?

Reply:
Well this is something that I could check in the IsIncluded function.
But currently this function only check if all or a dedicated resource is included, not if the resource is not in exclude.

So we need to check if a user set include `aws_iam_group_membership` and exclude `aws_iam_group_membership`. If both are set, the resource will not be imported